docs/SubmittingPatches.md

```diff
@@ -30,6 +30,19 @@ Certificate of Origin" located at the bottom of this page:
   see this [link]
   (https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).
 
+## Release notes
+
+To facility creating release notes at release time, all non-trivial
+changes must include a release note snippet with the pull request.
+This can be either a separate commit, or as part of a single commit
+(generally, if there are multiple commits to the pull request, the
+release note snippet should be a separate commit, and a pull request
```

Comment:
I agree on separate commit but separate PR should not be mandatory. I prefer to have release notes with the PR that introduces changes, when possible.

Reply:
I can try to clarify this, because I agree the notes should definitely be part of the same PR.